v2/rscg_examples_site/docs/RSCG-Examples/TrimItEasy.md

```
@@ -0,0 +1,529 @@
+Phong Nguyen 
+![Alt text](https://github.com/phongnguyend.png)
+:::
+
+## Original Readme
+:::note
+
+### TrimItEasy
+
+A simple and efficient .NET library for automatically trimming string properties in complex objects.
+
+###### Installation
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h4; Actual: h6

(MD001, heading-increment)